src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -372,7 +422,7 @@ private String getVariantDataString(final VariantContext variant) {
         );
     }
 
-    private String getVariantInfoString(final VariantContext variant) {
+    private String getVariantInfoStringOld(final VariantContext variant) {
```

Comment:
This method appears to be unreferenced now.

Reply:
removed